docs/reference/errors-and-warnings/NU1004.md

```diff
@@ -0,0 +1,46 @@
+---
+title: NuGet Error NU1004
+description: NU1004 error code
+author: nkolev92
+ms.author: nikolev
+ms.date: 01/02/2022
+ms.topic: reference
+f1_keywords: 
+  - "NU1004"
+---
+
+# NuGet Error NU1004
+
+## Scenario 1
+
+<pre>The project target frameworks are different than the lock file's target frameworks. Lock file target frameworks: net6.0,net5.0. Project target frameworks net6.0.
+The packages lock file is inconsistent with the project dependencies so restore can't be run in locked mode. Disable the RestoreLockedMode MSBuild property or pass an explicit --force-evaluate option to run restore to update the lock file.</pre>
+
+### Issue
+
+When restore is run in locked mode, it ensures that the project dependencies have not changed since the last restore.
+A new framework, `net5.0` has been added to the project.

```

Comment:
I don't think there are, but are there any scenarios where running something like `dotnet run -f net5.0` or `dotnet build -f net5.0`, or `dotnet publish -f net5.0` might cause this error to be reported?  Basically, any scenario where telling the dotnet cli to use a specific RID when the project itself supports multiple?

If so, should we call out this scenario specifically?

Reply:
This shouldn't be happening with net5.0. 

Lock files + rids are challenging though for sure.